### PR TITLE
Changelog for 1279

### LINKED ISSUE #1279: When ZOWE_HOST_NAME cannot be resolved default to ZOWE_IP_ADDRESS in instance.env
**Describe the bug**
On some systems the ZOWE_HOST_NAME cannot be resolved externally.  It is used in the configure.sh scripts for the desktop exolorers, and this causes 404 errors

**Expected behavior**
When the ZOWE_HOST_NAME cannot be resolved if none is specified as an override then default to the ZOWE_IP_ADDRESS so that the instance.env file ZOWE_HOST_NAME is the ZOWE_IP_ADDRESS

This logic will be zowe-init.sh.  

**Screenshots**
When parsing the results of the dig command on the system in error is shown below

image (12)

The result of running zowe-init.sh on the system in error is shown belos
image (13)

**Additional context**
This problem will resolve https://github.com/zowe/explorer-jes/pull/169

### PR DESCRIPTION
**Pull request by Joe-Winchester**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Necessary documentation (if appropriate) have been added / updated
- [x] DCO signoffs have been added to all commits, including this PR

#### PR type
- [ ] Bugfix 
- [ ] Feature 
- [x] Other...CHANGELOG.md update

#### Relevant issues
Fixes #1279

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Change Log
+
+All notable changes to the Zowe Installer will be documented in this file.
+
+## Recent Changes
+
+- When `localhost` cannot be resolved use the IP address instead.
```
Comment (John-A-Davies) [inline CHANGELOG.md:7]: I think it's better to say `hostname` than `localhost`.  The `localhost` is an alias for the loopback address 127.0.0.1, but `hostname` is a command that returns the name of the system on the network, e.g. S0W1.DAL-EBIS.IHOST.COM

Comment (Joe-Winchester) [reply CHANGELOG.md]: Tks for the feedback @John-A-Davies - I've made some updates so let me know if the latest push is any better.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,4 +4,4 @@ All notable changes to the Zowe Installer will be documented in this file.
 
 ## Recent Changes
 
-- When `localhost` cannot be resolved use the IP address instead.
\ No newline at end of file
+- When the hostname cannot be resolved use the IP address instead.  This covers the scenarios when `localhost` was not specified as an alias for the loopback address 127.0.0.1 as well as when the USS `hostname` command returned a system name that wasn't externally addressable, such as `S0W1.DAL-EBIS.IHOST.COM` which occurs on an image created from the z/OS Application Developers Controlled Distribution (ADCD).
```
Comment (John-A-Davies) [inline CHANGELOG.md:7]: AFAIK, localhost is not involved anywhere in this check in `zowe-init.sh`, so the wording 

> s when `localhost` was not specified as an alias for the loopback address 127.0.0.1 as well as

should be removed.

Comment (Joe-Winchester) [reply CHANGELOG.md]: Tks @John-A-Davies - I've pushed an updated to match the wording correction.

**Pull request merged**